README.md

```
@@ -68,6 +68,17 @@ deno task test:astro:unit    # Unit & component tests (Vitest)
 deno task test:astro:e2e     # End-to-end tests (Playwright)
 ```
 
+### Debugging Acceptance Test
+
+1. Run WebStorm "Attach to Node.js/Chrome" debugger
+2. Set breakpoints
+3. Run
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)